src/ClientProtos/preset.proto

```diff
@@ -6,18 +6,21 @@ syntax = "proto3";
 
 package orbit_client_protos;
 
+// This message is used for loading presets from
+// old format, do not use it to save presets.
 message PresetModule {
   repeated uint64 function_hashes = 1;
   repeated uint64 frame_track_function_hashes = 2;
 }
 
+message PresetModuleInfo {
+  repeated string function_names = 1;
+  repeated string frame_track_function_names = 2;
+}
+
```

Comment:
Could we have added the new fields in "PresetModule" so that we'd have had the same code path for both the old and new format? 

Something like: 

message PresetModule {
  repeated uint64 function_hashes = 1;
  repeated uint64 frame_track_function_hashes = 2;
  repeated string function_names = 3;
  repeated string frame_track_function_names = 4;
}

And then we eventually deprecate 1 and 2?

Reply:
In which case it could be that one module has combination of hashes and function names next one only function names and the last one only hashes. I think it is a bit clearer when we separate it on the top level. We do not want to accidently end up with mixed file format.

Reply:
ok sounds good to me